src/MRA.Jobs.Client/Pages/Client/Jobs.razor

```diff
@@ -1,11 +1,304 @@
 ﻿@page "/jobs"
 @layout Main
 @inherits LayoutComponentBase
+@using MRA.Jobs.Application.Contracts.JobVacancies.Responses
+@using MRA.Jobs.Application.Contracts.VacancyCategories.Responses;
+@inject IVacancyService VacancyService
+@inject ICategoryService CategoryService
+@implements IDisposable
 
-<div class="container">
-	<h3>Jobs</h3>
-</div>
+<head>
+    <style>
+        .categories {
+            width: 38%;
+            margin-right: 20px;
+        }
+
+        .vacancy {
+            width: 100%;
+        }
+
+        .card-vacancy:hover {
+            box-shadow: 2px 5px 10px 8px rgb(242 242 242 / 69%) !important;
+            transition: .2s;
+        }
+
+        .card-vacancy {
+            transition: all .2s ease-in-out;
+        }
+
+            .card-vacancy:hover h2 {
+                color: #0e6df9 !important;
+            }
+
+        .category {
+            transition: all .2s ease-in-out;
+        }
+
+        .selectedCategory {
+            background-color: #7aaffc;
+        }
+
+        .category:hover {
+            background-color: #7aaffc;
+        }
+
+        .category {
+            cursor: pointer;
+            border-radius: 5px;
+        }
+
+        .category-vacancies-value {
+            color: gray;
+        }
+
+        .page-index-selection {
+            margin-left: 0.5rem;
+            margin-right: 0.5rem;
+        }
+
+        .page-rel-selection {
+            margin-left: 0.3rem;
+            margin-right: 0.3rem;
+        }
+
+        .reset-input-btn {
+            border: none;
+        }
+    </style>
```

Comment:
it should be in jobs.razor.css

Reply:
Couldnt do that because of the bug that didnt let to work css styles in separate file.